core/trino-main/src/main/java/io/trino/sql/planner/ConnectorExpressionTranslator.java

```diff
@@ -483,6 +503,20 @@ public SqlToConnectorExpressionTranslator(Session session, Map<NodeRef<Expressio
             throw new UnsupportedOperationException("Unsupported sign: " + node.getSign());
         }
 
+        @Override
+        protected Optional<ConnectorExpression> visitCoalesceExpression(CoalesceExpression node, Void context)
+        {
+            ImmutableList.Builder<ConnectorExpression> arguments = ImmutableList.builderWithExpectedSize(node.getOperands().size());
+            for (Node argument : node.getChildren()) {
+                Optional<ConnectorExpression> translated = process(argument);
```

Comment:
@wendigo 's PR adds some shorthand function for this, am i right, @wendigo ?

no change requested now, but we should unify later

Reply:
Yes,I will note it down to unify that after those PRs are merged.